django/contrib/postgres/signals.py

```diff
@@ -5,10 +5,13 @@
 from psycopg2.extras import register_hstore
 
 from django.db import connections
+from django.db.backends.base.base import NO_DB_ALIAS
 
 
 @functools.lru_cache()
 def get_hstore_oids(connection_alias):
+    if connection_alias == NO_DB_ALIAS:
```

Comment:
maybe better to just do `if connection_alias not in connections` ? This feels like it's jumping some encapsulation. Could also be done in `register_type_handlers` to avoid duplication

Reply:
I'm not sure about the `if connection_alias not in connections` proposal as handling no_db explicitly seems less likely to hide errors. Putting the check in `register_type_handlers()` seems better.

Reply:
Fair point